pimd/pim_zebra.c

```diff
@@ -541,45 +541,6 @@ void pim_zebra_upstream_rpf_changed(struct pim_instance *pim,
 	pim_upstream_update_join_desired(pim, up);
 }
 
-static void scan_upstream_rpf_cache(struct pim_instance *pim)
```

Comment:
@donaldsharp do you see any value in keeping this function around? I know you dropped the single use case for it in the current code, but 'm not sure if it would be applicable in other future use cases.

Reply:
I do not believe it is applicable anymore.  All relevant upstream data structures are traversed when we get a nexthop change.  As such we do not need to do this anymore.